### ISSUE TITLE
Chai-should typeof date produces bad transform

### ISSUE THREAD

**Issue by saveman71** [skovhus/jest-codemods#173]
Repro:

```js
'use strict';

const expect = require('chai').expect;

describe('Gift', () => {
  test('should be a date', () => {
    expect(new Date()).to.be.a('Date');
  });
});
```

produces

```js
'use strict';

describe('Gift', () => {
  test('should be a date', () => {
    expect(typeof new Date()).toBe('Date');
  });
});
```

`typeof new Date()` is `"object"`, thus the test fails.

**Comment by skovhus** [comment]
Good catch, thanks for reporting this. Let me know if you would like to submit a PR fixing this issue.

**Comment by skovhus** [comment]
Released as jest-codemods@0.21.0

https://github.com/skovhus/jest-codemods/releases/tag/0.21.0
